discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -123,7 +123,11 @@ def get_operators(self):
         return self.api_client.get_cluster_operators(self.id)
 
     def generate_image(self):
```

Comment:
I think that this method should be on infra_env and not in cluster

Reply:
created a generate_infra_env method and added a deprecation warning